cdap-cli/src/main/java/co/cask/cdap/cli/command/DeployAppCommand.java

```diff
@@ -51,13 +51,15 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     File file = resolver.resolvePathToFile(arguments.get(ArgumentName.APP_JAR_FILE.toString()));
     Preconditions.checkArgument(file.exists(), "File " + file.getAbsolutePath() + " does not exist");
     Preconditions.checkArgument(file.canRead(), "File " + file.getAbsolutePath() + " is not readable");
-    applicationClient.deploy(cliConfig.getCurrentNamespace(), file);
+    String appConfig = arguments.get(ArgumentName.RUNTIME_ARGS.toString(), "");
+    applicationClient.deploy(cliConfig.getCurrentNamespace(), file, appConfig);
     output.println("Successfully deployed application");
   }
 
   @Override
   public String getPattern() {
-    return String.format("deploy app <%s>", ArgumentName.APP_JAR_FILE);
+    return String.format("deploy app <%s> [<%s>]", ArgumentName.APP_JAR_FILE,
+                         ArgumentName.RUNTIME_ARGS);
```

Comment:
its not really runtime args, maybe create a new argument name for application config?